compiler/rustc_feature/src/active.rs

```diff
@@ -148,8 +148,7 @@ macro_rules! declare_features {
 // was set. This is most important for knowing when a particular feature became
 // stable (active).
 //
-// Note that the features are grouped into internal/user-facing and then
```

Comment:
They are split into several categories, including internal, aren't they?

Reply:
They are, but the ordering doesn't take that into account. It used to, but at some point it was changed to be purely alphabetical. So the old comment was wrong.

Reply:
https://github.com/rust-lang/rust/pull/116550/commits/7764b66368de7b7f2b8572cee70fce91dc8e1e5b#diff-a82bdbaaa3c79096a30755dce8896aaf2885128a7f3f3a290072b767f05b6095L233 this is starting with a in the second group

Reply:
Right, my mistake. I have removed that commit.